### ISSUE TITLE
Photopea is changing colors after just uploading

### ISSUE THREAD

**Issue by FrAPo777** [photopea/photopea#6042]
Hi I have a strange issue. Photos are taken on iPhone 13 Pro. After simply uploading photos without any adjustments, Photopea will slightly change the colors by itself. I am attaching the photos without any editing, only after uploading.

The first photo is before uploading and the second is after uploading.

![IMG_8770](https://github.com/photopea/photopea/assets/145232691/08f0b0e2-af3e-4977-9c4b-c260f0035be4)
![IMG_8770a](https://github.com/photopea/photopea/assets/145232691/fa3d24a6-b704-4f2f-b404-a39e8a9b80ba)

Anyone know why this is happening on its own?
Thank you

**Comment by photopea** [comment]
Hi, I can not see any difference in your two photos. Could you send us a JPG file exactly as it was taken by your phone? Exact ones-and-zeros? Maybe put it into a ZIP, so that Github does not modify it in any way.

**Comment by FrAPo777** [comment]
The difference is very small but crucial for me. Concentrate on the white of the car. The best way to see the difference is to download the two photos and quickly click between them.

I am sending 2 more photos of the interior. Again the filter seems to have changed after uploading.

Sending photos in ZIP.
[Octavia RS 2.zip](https://github.com/photopea/photopea/files/12643016/Octavia.RS.2.zip)
[Octavia RS.zip](https://github.com/photopea/photopea/files/12643017/Octavia.RS.zip)

**Comment by photopea** [comment]
Each of your ZIPs contains two photos. Which one of them is the original photo taken by your phone? I just need one, original photo.

BTW. could you start a new issue, use the issue template, and answer the questions and write down steps? It would really save us hours of discussing.
